ntp-daemon/src/config/mod.rs

```diff
@@ -205,6 +206,13 @@ pub enum ConfigError {
 
 impl Config {
     async fn from_file(file: impl AsRef<Path>) -> Result<Config, ConfigError> {
+        let meta = std::fs::metadata(&file).unwrap();
+        let perm = meta.permissions();
+
+        if perm.mode() & libc::S_IWOTH == 0o002 {
```

Comment:
```suggestion
        if perm.mode() & libc::S_IWOTH != 0 {
```
Removes magic number